### PR TITLE
Feature/dockerize app

### PR DESCRIPTION
**Pull request by ptatarsk**
Done:

- [x] add dockerfile for client
- [x] add dockerfile for server and composee for db, server, client
- [x] correct ports in docker containers/composer, add volume for db
- [x] fix npm scripts, some naming

### CODE REVIEW

File: client/.dockerignore [hunk 1]
```diff
@@ -0,0 +1,9 @@
+build
+node_modules
+Dockerfile
+.dockerignore
```
Comment (imbaniac) [inline client/.dockerignore:4]: you've added .dockerignore to .dockerignore)

File: server/.dockerignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+node_modules
+Dockerfile
```
Comment (imbaniac) [inline server/.dockerignore:2]: same as upper comment

File: server/configs/db.config.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-  "url": "postgres://klseamrk:DjagK4U_8J9ylYEwdrOUS-tI5rwRXUc6@queenie.db.elephantsql.com:5432/klseamrk"
+  "url": "postgres://mydb:mydb@db:5432/mydb"
```
Comment (imbaniac) [inline server/configs/db.config.json:2]: You should access those constants from process.env (thanks to docker-compose)

File: server/index.js [hunk 4]
```diff
@@ -3,6 +3,7 @@ const dbConfig = require('./configs/db.config.json');
 const jwtConfig = require('./configs/jwt.config.json');
 
 const { url } = dbConfig;
+console.log('url: ', url);
```
Comment (imbaniac) [inline server/index.js:6]: remove console.log

File: server/index.js [hunk 5]
```diff
@@ -31,7 +32,7 @@ fastify
   .register(require('./routes/registration'))
   .register(require('./routes/timeslot'));
 
-fastify.listen(3300).catch((err) => {
+fastify.listen(3300, '0.0.0.0').catch((err) => {
```
Comment (imbaniac) [inline server/index.js:35]: user port from process.env (from docker-compose)

File: server/index.js [hunk 6]
```diff
@@ -31,7 +32,7 @@ fastify
...
   console.log('aaaaaa');
```
Comment (imbaniac) [inline server/index.js:36]: remove

**Review by imbaniac — CHANGES_REQUESTED** [review]

File: client/.dockerignore (hunk 1 above)
Comment (ptatarsk) [reply client/.dockerignore]: Done

File: server/.dockerignore (hunk 2 above)
Comment (ptatarsk) [reply server/.dockerignore]: Done

File: server/configs/db.config.json (hunk 3 above)
Comment (ptatarsk) [reply server/configs/db.config.json:2]: Done in branch Feature/fix server issues

File: server/index.js (hunk 4 above)
Comment (ptatarsk) [reply server/index.js]: Done

File: server/index.js (hunk 5 above)
Comment (ptatarsk) [reply server/index.js]: Done

File: server/index.js (hunk 6 above)
Comment (ptatarsk) [reply server/index.js]: Done

**Review by imbaniac — APPROVED** [review]

**Pull request merged**